app/policies/resource_policy.rb

```diff
@@ -6,15 +6,22 @@ class ResourcePolicy < ApplicationPolicy
 
   # Any user with an account can create or update resources
   def create?
-    true
+    return false unless user.active?
+    return true if user.full_access?
+
+    # Collection-limited users can only update DROs
+    return false unless record.dro?
```

Comment:
I'm not sure I get this. We wanted to allow all active users to create collections, right? Should this be `return true if record.collection?`

Reply:
I'd suggest that in the simplest possible use case, a restricted user can only deposit to a single collection, not create collections. That is certainly the case with the one restricted user that we know about, Google Books. Once we understand other use cases, we can extend the authorization implementation.